tst/com/amazon/corretto/crypto/provider/test/EvpSignatureSpecificTest.java

```diff
@@ -460,25 +625,52 @@ public void simpleCorrectnessAllAlgorithms() throws Throwable {
             final Signature nativeSig = Signature.getInstance(algorithm, AmazonCorrettoCryptoProvider.INSTANCE);
             final Signature bcSig = Signature.getInstance(bcAlgorithm, TestUtil.BC_PROVIDER);
 
-            try {
-                // Generate with native and verify with BC
-                nativeSig.initSign(pair.getPrivate());
-                bcSig.initVerify(pair.getPublic());
-                nativeSig.update(message);
-                bcSig.update(message);
-                byte[] signature = nativeSig.sign();
-                assertTrue(bcSig.verify(signature), "Native->BC: " + algorithm);
-
-                // Generate with BC and verify with native
-                nativeSig.initVerify(pair.getPublic());
-                bcSig.initSign(pair.getPrivate());
-                nativeSig.update(message);
-                bcSig.update(message);
-                signature = bcSig.sign();
-                assertTrue(nativeSig.verify(signature), "BC->Native: " + algorithm);
-            } catch (SignatureException ex) {
-                throw new AssertionError(algorithm, ex);
-            }
+            simpleCorrectnessSignVerify(algorithm, pair, bcSig, nativeSig);
+        }
+
+        // NOTE: for RSASSA-PSS, test supported digest lengths, but  keep PSS and MGF1 digest lengths equal because
+        //       BouncyCastle doesn't support differing lengths. we test differing lengths exhaustively against
+        //       SuncJCE in EvpSignatureTest. enforce min key size of 2048 bits, consistent with ffSize above.
+        for (int shaVersion : new int[] {1, 224, 256, 384, 512}) {
+            final String algorithm = "RSASSA-PSS";
```

Comment:
Add a check which skips over these tests for versions < 2.0.0

Reply:
added here and in EvpSignatureTest parameter generation.